test/inclusivity.jl

```diff
@@ -47,6 +47,9 @@
         @test hash(inc) == hash(cp)
         @test !isequal(inc, diff)
         @test hash(inc) != hash(diff)
+
+        # Verify that Inclusivity is treated as a scalar during broadcast
+        Inclusivity(false, true) .== Inclusivity(false, true)
```

Comment:
```suggestion
        @test size(Inclusivity(false, true) .== Inclusivity(false, true)) == ()
```

Reply:
Thanks for catching that! I can't believe I didn't see it was missing.